test.js

```diff
@@ -39,24 +53,24 @@ test('regexp support in the lookup', function (t) {
   t.plan(1)
 
   var instance = bloomrun()
-  var pattern = { cmd: 'save', prefs: 'userId' }
+  var pattern = { prefs: 'userId' }
   var payload = '1234'
 
   instance.add(pattern, payload)
 
-  t.deepEqual(instance.lookup({ cmd: 'save', prefs: /user.*/ }), payload)
+  t.deepEqual(instance.lookup({ prefs: /user.*/ }), payload)
```

Comment:
can you please restore the previous tests and add new ones? We need tests for both cases anyway.